src/docs/metrics/mean-realized-price/index.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Mean Realized Price
+author: Filip
+date: 2024-04-01
+description: Mean Realized Price Metrics
+---
+
+## Definition
+
+The Mean Realized Price metric calculates the average price at which tokens have been transacted 
+relative to their current circulating supply. It provides insights into the average profitability 
```

Comment:
If you can add some type of example, it would help a lot.
For example, we have this example for the sentiment metrics:
<img width="655" alt="image" src="https://github.com/santiment/academy/assets/6518376/74f173fa-c396-4c34-aca6-5072a5d29597">